pkg/reconciler/clusterstack/clusterstack.go

```diff
@@ -73,8 +81,36 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 	return nil
 }
 
-func (c *Reconciler) reconcileClusterStackStatus(clusterStack *v1alpha1.ClusterStack) (*v1alpha1.ClusterStack, error) {
-	resolvedClusterStack, err := c.ClusterStackReader.Read(clusterStack.Spec)
+func (c *Reconciler) reconcileClusterStackStatus(ctx context.Context, clusterStack *v1alpha1.ClusterStack) (*v1alpha1.ClusterStack, error) {
+	var keychain *authn.Keychain
+	var err error
```

Comment:
I don't think this extra err var is used

Reply:
You are right. Not anymore. It was a leftover. Thanks.